app/code/Magento/Sales/Model/Order/Creditmemo/Total/Shipping.php

```diff
@@ -56,12 +56,12 @@ public function collect(\Magento\Sales\Model\Order\Creditmemo $creditmemo)
         // amounts including tax
         $orderShippingInclTax = $order->getShippingInclTax();
         $orderBaseShippingInclTax = $order->getBaseShippingInclTax();
-        $allowedTaxAmount = $order->getShippingTaxAmount() - $order->getShippingTaxRefunded();
+        $allowedTaxAmount = $order->getShippingTaxAmount() + $order->getShippingDiscountTaxCompensationAmount() - $order->getShippingTaxRefunded();
```

Comment:
Can you please break this line into several lines (as it is done on lines 61-64) to ensure it's not braking the 120 characters limit

Reply:
Hi @sivaschenko
I have updated it.